src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -582,10 +582,11 @@ def add_electrical_series(
         warn(f"compression_opts ({compression_opts}) were passed, but compression type is 'lzf'! Ignoring options.")
         compression_opts = None
 
-    default_name = f"ElectricalSeries_{write_as}"
+    segment_name_signature = "" if checked_recording.get_num_segments() == 1 else segment_index
+    default_name = f"ElectricalSeries{segment_name_signature}_{write_as}"
```

Comment:
```suggestion
    default_name = f"ElectricalSeries{segment_name_signature}{write_as.capitalize()}"
```
@CodyCBakerPhD 
Let me know if this makes sense.

Reply:
I'd reverse the order - example output: `ElectricalSeriesRaw0`, `ElectricalSeriesProcessed3`. Again from https://nwbinspector.readthedocs.io/en/dev/best_practices/general.html#naming-conventions, though not stated explicitly but numbers likely ought to occur at the end of the name.

Reply:
I am fine with that convention but I would like to do it in another PR as it breaks some tests non related to this feature.